src/batou_ext/resources/oci-template.nix

```diff
@@ -68,4 +78,23 @@
       ];
     };
   };
+
+  # {% if component.backend == "podman" %}
+  systemd.services."podman-{{ component.container_name }}".serviceConfig = {
+    User = "{{ component.user }}";
+    RuntimeDirectory = "{{component.container_name}}";
+    # {% if component.sd_notify == "healthy" %}
+    Delegate = true;
+    NotifyAccess = "all";
+    # {% endif %}
+  };
+
+  # {% if component.sd_notify == "healthy" %}
+  systemd.services."podman-{{ component.container_name }}".wants = [ "linger-users.service" ];
+  users.users."{{ component.user }}".linger = true;
+  # {% endif %}
+  # {% if component.sd_notify == "conmon" %}
+  users.users."{{ component.user }}".linger = false;
```

Comment:
This means btw we can't have conmon & healthy for containers running as the same user, unfortunately :(